cluster-operator/src/test/resources/io/strimzi/operator/cluster/operator/assembly/CruiseControlJSON/CC-State-proposal-not-ready.json

```diff
@@ -0,0 +1 @@
+{"AnalyzerState":{"isProposalReady":false,"readyGoals":[]},"MonitorState":{"trainingPct":9.6,"trained":false,"numFlawedPartitions":0,"state":"RUNNING","numTotalPartitions":2065,"numMonitoredWindows":1,"monitoringCoveragePct":100.0,"reasonOfLatestPauseOrResume":"N/A","numValidPartitions":2065},"ExecutorState":{"state":"NO_TASK_IN_PROGRESS"},"AnomalyDetectorState":{"recentBrokerFailures":[],"recentGoalViolations":[],"selfHealingDisabled":["DISK_FAILURE","BROKER_FAILURE","GOAL_VIOLATION","METRIC_ANOMALY","TOPIC_ANOMALY"],"balancednessScore":100.0,"selfHealingEnabled":[],"recentDiskFailures":[],"metrics":{"meanTimeToStartFixMs":0.0,"numSelfHealingStarted":0,"ongoingAnomalyDurationMs":0,"numSelfHealingFailedToStart":0,"meanTimeBetweenAnomaliesMs":{"TOPIC_ANOMALY":0.0,"GOAL_VIOLATION":0.0,"DISK_FAILURE":0.0,"BROKER_FAILURE":0.0,"METRIC_ANOMALY":0.0}},"recentMetricAnomalies":[],"recentTopicAnomalies":[],"selfHealingEnabledRatio":{"DISK_FAILURE":0.0,"BROKER_FAILURE":0.0,"GOAL_VIOLATION":0.0,"METRIC_ANOMALY":0.0,"TOPIC_ANOMALY":0.0}},"version":1}
```

Comment:
Is there a reason we haven't inlined these JSON strings into a java class?

Reply:
These files are example return response for the Mock Cruise Control sever and are copies of the raw response from `curl <cc-endpoint> > file.json` and are returned verbatim by the mock server.